README.md

```diff
@@ -1,36 +1,14 @@
-## Apply for Radxa CM3
-
-Radxa is donating some free [ROCK3 computing modules](https://wiki.radxa.com/Rock3/CM3) to open source hardware projects.
-
-
-
-## How to apply
-
-Send a new PR to this repository on GitHub, append your application table. Radxa team will review/ask questions/discuss about the project detail. If the project is interesting or welcomed by the community, we will approve it. Radxa Team will contact the developer for the further sample shipping&development etc.
-
-
-
 ## CM3 projects list:
 
-| Project Name:     | TBD     |
-| ----------------- | ---- |
-| CM3 Model:        | WiFi, eMMC, 4GB+ RAM preferred     |
-| Project Hardware: | Currently the RPi CM4 + LCD    |
+| Project Name:     | Lemuria  |
```

Comment:
We should be mentioning about IIoT hardware, not software only.

Reply:
Incorporated the required changes.